packages/theme-chalk/src/text.scss

```diff
@@ -0,0 +1,57 @@
+@use 'sass:map';
+
+@use 'mixins/mixins' as *;
+@use 'mixins/var' as *;
+@use 'common/var' as *;
+
+@include b(text) {
+  @include set-component-css-var('text', $text);
+}
+
+@include b(text) {
+  display: inline-flex;
+  height: getCssVar('text', 'size');
```

Comment:
## Question

Is there a reason to have a fixed height value? The text area is likely to be long enough.